code/modules/games/tarot.dm

```diff
@@ -28,3 +28,8 @@
 		playsound(user, 'sound/items/cardshuffle.ogg', 50, 1)
 		user.visible_message("<span class='notice'>[user] shuffles [src].</span>", "<span class='notice'>You shuffle [src].</span>")
 		cooldown = world.time
+
+// This shuffles the deck, allowing you to do so without moving your mouse all the way to the top of the screen.
+/obj/item/deck/tarot/AltClick()
+	. = ..()
+	deckshuffle()
```

Comment:
You added this behavior to ```/obj/item/deck``` already, it will inherit deckshuffle() from it.

Also, this comment could be removed too.

Reply:
Removed.